include/miral/miral/window_manager_tools.h

```diff
@@ -163,6 +163,10 @@ class WindowManagerTools
     /// Find the active output area
     auto active_output() -> mir::geometry::Rectangle const;
 
+    /// Find the active zone area
+    /// \remark Since MirAL 3.0
+    auto active_zone() -> mir::geometry::Rectangle const;
```

Comment:
Wouldn't it make more sense to return a `miral::Zone`? The policy may want to match the returned zone to an existing application zone it knows about.

Also, can we change the name to `active_application_zone()`? That would clearly indicate that the returned value will be one of the application zones the policy knows about.